app/assess/templates/assessor_tasklist.html

```diff
@@ -5,9 +5,12 @@
 {% from 'macros/criteria_element.html' import criteria_element -%}
```

Comment:
This file in particular has a lot of conditional logic in the Jinja templates, I think we'd be more confident in this logic if we had some test coverage, would also be easier to develop on in the future I think

Reply:
great point, added tests 👍🏾